src/embed.ts

```diff
@@ -214,6 +215,7 @@ export default async function embed(
         .attr('href', '#')
         .on('click', () => {
           post(window, editorUrl, {
+            config: config ? config : null,
```

Comment:
Why not just config or config || null?

Reply:
I meant if the config is undefined then send it null. `config || null` would do the same and is better. Changing it.

Reply:
@domoritz This is done.